examples/data/v6/demo/authorities/authority-v1.0.0.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Out of curiosity: Did you find any differences in the mappings between 5 and 6?

Reply:
Yes, I found a couple of differences:
- instead of "type": "string" we should use "type": "text";
- the "index" field should be a boolean.

https://www.elastic.co/blog/strings-are-dead-long-live-strings
According to this blog post, these differences were introduced with elasticsearch5, but with elasticsearch6 there is no backward compatibility, so they are mandatory.

Reply:
To add to this, one "major" change is the [removal of mapping types](https://www.elastic.co/guide/en/elasticsearch/reference/6.2/removal-of-types.html). The norm now is to set the `doc_type` for any mapping to always be  `_doc` (so [`ad-v1.0.0`](https://github.com/inveniosoftware/invenio-marc21/blob/master/invenio_marc21/mappings/v5/marc21/authority/ad-v1.0.0.json#L7) should be just `_doc`).

This doesn't really affect us since all of our mappings have only one document type... We might have to check out how this affects `elasticsearch-py` since the `doc_type` paremter for calling e.g. `current_search.index(index='records', doc_type=..., ...)` might be different or simpllified.